src/main.py

```diff
@@ -190,6 +191,15 @@ def accentless_sort(value):
   return sorted(value, key=lambda i: strip_accents(i[1]).lower())
 
 
+def get_featured_quotes(lang, year):
+    featured_quotes = {}
+    featured_quote_filename = 'templates/%s/%s/featured_chapters.json' % (lang, year)
```

Comment:
Any particular reason why we use `%s` against `.format()` ?

Reply:
To be perfectly honest I wasn't even aware [this was a thing!](https://realpython.com/python-string-formatting/) (I'm not a Python developer by day!). It looks like this repo [originally started on Python 2](https://github.com/HTTPArchive/almanac.httparchive.org/issues/48) so maybe that's the reason?

Anyway, that's the convention currently used in this code base so let's not make this one line out of sync. If we want to change this, then we should do it in a separate PR across the code base - though not seeing a pressing need to do this.

Reply:
> It looks like this repo originally started on Python 2 so maybe that's the reason?

Makes sense

> Anyway, that's the convention currently used in this code base so let's not make this one line out of sync. If we want to change this, then we should do it in a separate PR across the code base - though not seeing a pressing need to do this.

Agreed